Sources/LLVM/Constant.swift

```diff
@@ -126,6 +183,40 @@ public struct Constant<Repr: ConstantRepresentation>: IRValue {
     }
   }
 
+  /// Creates a constant negate operation to negate a value.
+  ///
+  /// - parameter lhs: The operand to negate.
+  ///
+  /// - returns: A constant value representing the negation of the given constant.
+  public static func negate(_ lhs: Constant<Floating>) -> Constant<Floating> {
+    return Constant<Floating>(llvm: LLVMConstFNeg(lhs.llvm))
+  }
+}
+
+extension Constant where Repr == Signed {
+
+  /// Creates a constant negate operation to negate a value.
+  ///
+  /// - returns: A constant value representing the negation of the given constant.
+  public func negate() -> Constant {
```

Comment:
Nit: `negated()`?

Swift API design guidelines and all...

Reply:
@vdka This one still stands.